spec/lib/bamboozled/tabular_spec.rb

```diff
@@ -0,0 +1,23 @@
+require_relative '../../spec_helper'
+require 'pry'
+
+describe "Employees" do
+
+  before do
+    @client = Bamboozled.client(subdomain:'x', api_key:'x')
+  end
+
+  describe "#add_position" do
+    it "adds a position row to the specified table in BambooHR" do
+      xml = YAML.load_file('spec/fixtures/add_row_xml.yml')
+      response = File.new('spec/fixtures/add_row_response.json')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.